package.json

```diff
@@ -77,7 +78,6 @@
     "ember-seamless-immutable-shim": "^1.0.0",
     "ember-sinon": "0.5.1",
     "ember-sortable": "^1.8.2",
-    "ember-tooltips": "0.7.0",
```

Comment:
Removing this may have broke the drop-downs in the app bar. Can you confirm? If this didn't break that then :+1: